bigframes/core/compile/compiled.py

```diff
@@ -351,7 +356,15 @@ def _to_ibis_expr(
         return table
 
     def filter(self, predicate: ex.Expression) -> UnorderedIR:
-        if any(map(is_window, map(self._get_ibis_column, predicate.unbound_variables))):
+        if any(
+            map(
+                is_window,
+                map(
```

Comment:
This if expression looks super long. Shall we just use a plain-old for loop?

for col in predicate.column_references:
  if (is_window(self._get_ibis_column(col.sql))):
      return ....

Reply:
yeah, went a bit overzealous with the nested maps, converted into loop